src/bots/ping/tests/handler.test.js

```diff
@@ -0,0 +1,11 @@
+import client from '../../../client';
+import handler from '../handler';
+
+describe('ping event', () => {
+  test('ping handler is called when ping event is emitted', () => {
+    const logSpy = jest.spyOn(client.logger, 'debug');
+    // Give handler() a msg.channel.send() since none is available in the test env
```

Comment:
A little nitpicky, but I'd prefer to just move the object to a variable rather than have this comment. Something similar to:
```
const message = { channel: { send: (...args) => {} } };
```